homeassistant/components/influxdb/const.py

```diff
@@ -72,7 +72,7 @@
 EVENT_NEW_STATE = "new_state"
 DOMAIN = "influxdb"
 API_VERSION_2 = "2"
-TIMEOUT = 5
+TIMEOUT = 10 # seconds
```

Comment:
```suggestion
TIMEOUT = 10  # seconds
```
Black and flake8 want two spaces before an inline comment. Make this and it should be good to go.